app/src/main/java/com/example/hangouts/homeScreen/MainActivity.java

```diff
@@ -11,4 +15,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
     }
+
+    // Temporary logout
```

Comment:
Its just a temporary logout button for testing, if you change something in the database, and you are still logged in it can cause pretty strange behavior.